packages/web/src/components/CheckboxField/_theme.scss

```diff
@@ -0,0 +1,24 @@
+
+$label-typography: tokens.$body-medium-text-regular;
+$label-color-default: tokens.$text-secondary-default;
+$label-color-disabled: tokens.$text-secondary-disabled;
+$label-required-color: tokens.$emotion-danger-default;
+
+$input-width: 18px;
+$input-typography: tokens.$body-medium-text-regular;
+$input-border-color-error: tokens.$emotion-danger-default;
+$input-checked-background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAATCAYAAACdkl3yAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAADySURBVHgBxdOxrcIwEAbg/xxe/SIBEiUNEtDgEWAERmADNmAFsgFMwAgwgstINCmRAAl6yHHniqAggkHib2Jbuk++yEdx17YNYwrCP0LCOOeEec0YrIhhERoCIsaA6j3L+EIMvpRQKJN/M2GC+wTK/hijY+oWgi1DIY/sUpc1+9YSYRYCFZAcWMuN4jLI9+2/byIFiHMk2rcWPGAvkQJEBrNmx1otuMMqIb7+4UGezAWj/da5loyOHlRByqACppsqyDPIY1fGWBeRzOIrRFN7ch5HJLfQVJzEn89aCURI8GGk+4R00ejaoWzaCIgA2SF1mxutP3ra7Lq18QAAAABJRU5ErkJggg==');
+$input-checked-disabled-background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAABpSURBVHgB7ZRbDYAwDEV7NwRMAkLmBQ04AAeIWlIZzAET0AS6EAwUwldP0tfPSb8umHkSkQ1AIhs1xriglLLrMdI72vBIcs4gA/rIqSMF+ggXuchF/4h6jBw6rVl0S4AatM19ITtNa70AVWIXOaxEgCcAAAAASUVORK5CYII=');
+$input-unchecked-background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAABsSURBVHgB7dQhEkBAFIfx7yFJxhZRRHo3cg7NERxNE6mCsTdYW8zIy0j7tVd+809PTKM9CROOgoAcbOIYpWx1Fah5l81u5FhmISDTqh9FkfBREYpQhP6BxHR6hv6iR1uam2pHUH+EYYL1Q4YL570Tjo01Xf0AAAAASUVORK5CYII=');
+$input-unchecked-disabled-background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAABpSURBVHgB7ZRbDYAwDEV7NwRMAkLmBQ04AAeIWlIZzAET0AS6EAwUwldP0tfPSb8umHkSkQ1AIhs1xriglLLrMdI72vBIcs4gA/rIqSMF+ggXuchF/4h6jBw6rVl0S4AatM19ITtNa70AVWIXOaxEgCcAAAAASUVORK5CYII=');
+$input-indeterminate-background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAB1SURBVHgB7dQpDoAwEEDRPwWFItQgkeB6I86B4wgcDYcEiyDhBsMiMZCyKJ5rxc8kk4zY3JUYGpQYDwqDKLUkhesFMu6ZzQORTWx4yB86Fx4/pq4VLrCFU96YSI5lX//6Pw0JM/cNQWTTcY259eF12PZBlGoBaA0Tf51eb4QAAAAASUVORK5CYII=');
+$input-indeterminate-disabled-background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAABwSURBVHgB7dSxDYAgEAXQ/8EBGMFB2MUZ3EA3cCgSxpARGIBEj5ZGA2jFKwhQ/Fxyl6P3fkkpHSQN6gSt9Ubn3CmPGW2i6hCSGYVORtCzqfyw1hIvyNhc+KIilsm1Rvt/DopoJLssKDnWfEG9XMh+A7STGCk4dTcbAAAAAElFTkSuQmCC');
+
+$message-typography: tokens.$body-medium-text-regular;
+$message-color-default: tokens.$text-secondary-default;
+$message-color-disabled: tokens.$text-secondary-disabled;
+$message-color-error: tokens.$emotion-danger-default;
+$message-spacing: tokens.$space-800;
```

Comment:
I would maybe prefer something like `$message-offset`, but I am not sure.

Reply:
Renamed to `offset`.